chaining-methods/challenges-10.test.js

```diff
@@ -0,0 +1,202 @@
+

```

Comment:
Make sure you check out `master` before you check out your new branch each day, to avoid getting old challenges in your new branch!